pkg/log/datadog.go

```diff
@@ -0,0 +1,13 @@
+package log
+
+type DatadogLogger struct {
+	logger Logger
+}
+
+func NewDatadogLogger(logger Logger) *DatadogLogger {
+	return &DatadogLogger{logger: logger}
+}
+
+func (d DatadogLogger) Log(msg string) {
+	d.logger.Error(msg)
```

Comment:
```suggestion
	d.logger.Debug(msg)
```

Reply:
Made the changes @rtribotte . Please review it again.